scipy/stats/_distribution_infrastructure.py

```diff
@@ -3555,7 +3562,17 @@ def make_distribution(dist):
         ``sample`` method must accept ``shape`` by position or keyword, but contrary
         to the public method of the same name, the argument it receives will be the
         *full* shape of the output array - that is, the shape passed to the public
-        method prepended to the broadcasted shape of random variable parameters.
+        method prepended to the broadcasted shape of random variable parameters. If
+        multiple parameterizations are defined, these methods must accept all
+        parameters from all parameterizations.
+
+        If the class's ``parameters`` attribute contains a ``tuple`` defining multiple
+        parameterizations, then the class must also define a ``__process_parameters``
```

Comment:
```suggestion
        parameterizations, then the class must also define a ``_process_parameters``
```
We *could* allow the users to make this a public function (`process_parameters`), instead.